features/admin.users.v1/api/users.ts

```diff
@@ -470,3 +470,61 @@ export const terminateAllUserSessions = (userId: string): Promise<AxiosResponse>
                 error.config);
         });
 };
+
+/**
+ * Interface for the resend code request payload.
+ */
+export interface ResendCodeRequest {
```

Comment:
Let's move this to `models/` folder

Reply:
Resolved via https://github.com/wso2/identity-apps/pull/7992/commits/fb3422572ea4918d0817c4d3802d2137fa0dd495